apps/explorer/src/components/orders/BridgeDetailsTable/RefundStatus.tsx

```
@@ -0,0 +1,69 @@
+        </StatusWrapper>
+      )
+
+    case RefundStatusEnum.FAILED:
+      return (
+        <StatusWrapper>
+          <StatusLabel status="failed" />
+          <span>Refund failed</span>
+        </StatusWrapper>
+      )
+
+    case RefundStatusEnum.NOT_INITIATED:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)